src/implementations/config/create.ts

```diff
@@ -35,9 +37,17 @@ export const create = async ({
 
   const hasTransferableRecord = forms.some((form) => form.type === "TRANSFERABLE_RECORD");
   const hasDocumentStore = forms.some((form) => form.type === "VERIFIABLE_DOCUMENT");
-  const hasDid = forms.some(
-    (form) => form.defaults.issuers.some((issuer) => issuer.identityProof?.type.includes("DID")) // TODO: v3 does not have `issuers` but `issuer` instead
-  );
+  const hasDid = forms.some((form) => {
+    //check form for v2/v3
+    if (isV3Form(form.defaults)) {
+      const v3Defaults = form.defaults as v3.OpenAttestationDocument;
+      const didCheckList = ["DID", "DNS-DID"];
```

Comment:
`utils.isRawV3Document(form.defaults)` will that work? -> https://github.com/Open-Attestation/open-attestation/blob/master/src/shared/utils/guard.ts#L30

Reply:
haha ok will update, forgot about that.